cmd/dep/testdata/harness_tests/init/manifest-exists/final/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Empty manifest in final, thought it was needed if we pass the exception test will double check and remove if not needed.